types/react-instantsearch-dom/react-instantsearch-dom-tests.tsx

```diff
@@ -7,9 +7,8 @@ import { Hit, connectRefinementList, connectMenu } from 'react-instantsearch-cor
 () => {
   const App = () => (
     <InstantSearch
-      appId="latency"
-      apiKey="3d9875e51fbd20c7754e65422f7ce5e1"
```

Comment:
We should keep at least one test that use this notation to check that it's actually working.